MinimedKit/PumpManager/MinimedPumpManager.swift

```diff
@@ -1147,6 +1147,15 @@ extension MinimedPumpManager: CGMManager {
     public var glucoseDisplay: GlucoseDisplayable? {
         return recents.sensorState
     }
+    
+    public var cgmStatus: CGMManagerStatus {
+        return CGMManagerStatus(hasValidSensorSession: hasValidSensorSession)
+    }
+    
+    public var hasValidSensorSession: Bool {
+        // No tracking of session available
+        return true
+    }
```

Comment:
Might be nicer to have an `unknown` returned then `yes there is a valid sensor session`.

Also, `sensor session` seems pretty specific to Dexcom's implementation. I take it to mean you want to know if the sensor is valid and it is currently reporting glucose measurements. Would `inMeasurementSession` or `activelyTakingMeasurements` (or similar) be more appropriate?

Reply:
The naming was chosen because we want to include the warmup period.  That rules out a lot of terms that indicate actively reporting of values, as values are not reported during warmup.  

The semantics that we want from this flag are that we're either getting values, or at least we expect values soon. Every CGM should be able to report on that semantic, so I don't want to add a third state that also needs to be handled.  My comment above is just that we currently aren't tracking any conditions that would return false in MDT implementation.